xpk/xpk.py

```diff
@@ -607,17 +613,23 @@ def get_zone():
   return zone_outputs[-1]  # The zone name lives on the last line of the output
 
 
-def zone_to_region(zone) -> str:
-  """Helper function converts zone name to region name.
+def get_zone_from_zone_or_region(name, cluster_type) -> str:
```

Comment:
Hmm I considering what other names we can pick here:

I like to describe the function by its purpose. This function will return the command needed to create a zonal or regional cluster. I think `get_cluster_type_argument(zone_name, cluster_type)` or something similar is clear to me. Let me know what you think!